ui/src/app/pages/workspace/workspace-edit-text.tsx

```diff
@@ -160,72 +160,72 @@ export const researchPurposeQuestions: Array<ResearchPurposeQuestion> = [
     header: <div> Please provide a summary of your research purpose by responding to the
       questions below.</div>,
     description: <div>Your responses should cover the major components of a research summary:
-      hypothesis, methods and anticipated findings. Therefore, please
+      hypothesis, methods, and anticipated findings. Therefore, please
       provide sufficiently detailed responses in plain language (without jargon), using as few
       technical terms as possible. </div>
   }, {
     header: 'What are the specific scientific question(s) you intend to study, and why is the ' +
-    'question important (i.e. relevant to science or public health)? \n',
+    'question important (i.e. relevance to science or public health)? \n',
     description: <div>If you are exploring the data at this stage to formalize a specific
       research question, please describe the reason for exploring the data, and the scientific
       question you hope to be able to answer using the data. <br/>
-      (Free text; 1000 Character limit)</div>
+      (Free text; 1000 character limit)</div>
```

Comment:
Row # 32

Reply:
I searched the whole code base for "0 character" to see if there were any other capitalizations like this for e.g. "50 characters"

Nope!  You got them all.